<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssetsManager</class>
 <widget class="QMainWindow" name="AssetsManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>559</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>База данных объектов</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTreeView" name="categoriesTree">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
          <horstretch>4</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::SelectedClicked</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectItems</enum>
        </property>
        <property name="expandsOnDoubleClick">
         <bool>false</bool>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QListWidget" name="assetsList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>6</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MainMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="MainMenu_Assets">
    <property name="title">
     <string>Объекты</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>Создать</string>
     </property>
     <addaction name="MainMenu_Assets_Create_GpuProgram"/>
    </widget>
    <addaction name="menu"/>
    <addaction name="separator"/>
    <addaction name="MainMenu_Assets_Import"/>
    <addaction name="separator"/>
    <addaction name="MainMenu_Assets_Exit"/>
   </widget>
   <addaction name="MainMenu_Assets"/>
  </widget>
  <widget class="QDockWidget" name="propertiesEditorSection">
   <property name="windowTitle">
    <string>Свойства</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="windowTitle">
    <string>Лог</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QPlainTextEdit" name="logTextArea">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="overwriteMode">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_2">
   <property name="text">
    <string>Геометрия</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Текстура</string>
   </property>
  </action>
  <action name="MainMenu_Assets_Import">
   <property name="text">
    <string>Импорт</string>
   </property>
  </action>
  <action name="MainMenu_Assets_Exit">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="MainMenu_Assets_Create_GpuProgram">
   <property name="text">
    <string>Шейдерная программа</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
